### ISSUE TITLE
Refactor "Photos others shared" and "Photos you shared"

### ISSUE THREAD

**Issue by derneuere** [LibrePhotos/librephotos#355]
We currently use react-pig to display the pictures in most of the views.

Two views that still use the old system are "Photos others shared" and "Photos you shared". This should be refactored to also use react-pig. This would also close the following issue #280

**Comment by jmalm** [comment]
Slowly progressing... I've now done the photos part of "Photos others shared". Should be easy to reuse in "Photos you shared". Next is the albums view.
I'm also fixing some problems with photo selections. May need to make adjustments in react-pig, not sure yet.

**Comment by jmalm** [comment]
Indeed, the selection problems required a change in react-pig: https://github.com/LibrePhotos/react-pig/pull/8
